packages/grid/src/vaadin-grid-column-mixin.d.ts

```diff
@@ -93,6 +93,16 @@ export declare class ColumnBaseMixinClass<TItem, Column extends ColumnBaseMixinC
    * - `column` The `<vaadin-grid-column>` element.
    */
   footerRenderer: GridHeaderFooterRenderer<TItem, Column> | null | undefined;
+
+  /**
+   * Custom part name for the header cell.
```

Comment:
Let's add JSDoc annotation for Lit plugin to recognise the dash cased attribute:

```
@attr {string} header-part-name
```

And same for the `footerPartName` property.

Reply:
Done